redirects-list.json

```diff
@@ -6,12 +6,13 @@
   "^/((?:deb|rpm)/.+)                       https://s3-us-east-2.amazonaws.com/dvc-s3-repo/$1 303",
   "^/(?:help|chat)/?$                       https://discordapp.com/invite/dvwXA2N 303",
   "^/(?:docs|documentation)(/.*)?$          /doc$1",
-  "^/doc/commands-reference(/.*)?$          /doc/command-reference$1",
+  "^/doc/?$                                 /doc/tutorials/get-started 307",
+  "^/doc/get-started                        /doc/tutorials/get-started",
   "^/doc/tutorial/?$                        /doc/tutorials",
   "^/doc/tutorial/(.*)?                     /doc/tutorials/deep/$1",
+  "^/doc/commands-reference(/.*)?$          /doc/command-reference$1",
   "^/doc/use-cases/data-and-model-files-versioning/?$ /doc/use-cases/versioning-data-and-model-files",
-  "^/doc/?$                                 /doc/get-started 307",
+  "^/doc/user-guide/contributing/?$         /doc/user-guide/contributing/core 307",
   "^/doc/understanding-dvc/?$               /doc/understanding-dvc/collaboration-issues 307",
-  "^/doc/changelog/?$                       /doc/changelog/0.18 307",
```

Comment:
I really want to get rid of changelog, make it an external link ... again a special case like a Home button ... we spend time on it again and again

Reply:
I thought it was an external link already before? I didn't introduce this in the Gatsby migration, just moved the order of some redirects.